wix/SourceDir/CreateSourceFolder.AdoptOpenJDK.ps1

```diff
@@ -1,10 +1,51 @@
+<#
+.SYNOPSIS
+    This script extracts the contents of a zip file to a directory structure that is expected by the Wix toolset.
+
+.DESCRIPTION
+    The script takes a zip file and extracts its contents to a directory structure that is expected by the Wix toolset.
+    The script also performs some cleanup on the extracted files.
+
+.PARAMETER openjdk_filename_regex
+    A regular expression that matches the OpenJDK filename. Default is ^OpenJDK(?<major>\d*).
+
+.PARAMETER jvm_regex
+    A regular expression that matches the JVM. Default is (?<jvm>hotspot|openj9|dragonwell).
+
+.PARAMETER jvm
+    The JVM to be used. If not provided, the script will attempt to extract the JVM from the filename.
+
+.PARAMETER platform_regex
+    A regular expression that matches the platform. Default is (?<platform>x86-32|x64|aarch64).
+
+.NOTES
+    File Name: CreateSourceFolder.AdoptOpenJDK.ps1
```

Comment:
Would probably prefer to use `EclipseAdoptium` or `Adoptium` but I suspect `AdoptOpenJDK` has meaning, or would break if we changed it or....

Reply:
That's what I suspected, ignore my request!